internal/controllers/ssp_controller.go

```diff
@@ -68,42 +69,50 @@ var kvsspCRDs = map[string]string{
 	"kubevirtcommontemplatesbundles.ssp.kubevirt.io": "KubevirtCommonTemplatesBundle",
 }
 
-// sspReconciler reconciles a SSP object
-type sspReconciler struct {
-	client           client.Client
-	uncachedReader   client.Reader
+// sspController reconciles a SSP object
+type sspController struct {
 	log              logr.Logger
 	operands         []operands.Operand
 	lastSspSpec      ssp.SSPSpec
 	subresourceCache common.VersionCache
 	topologyMode     osconfv1.TopologyMode
-	crdList          crd_watch.CrdList
 	areCrdsMissing   bool
+
+	client         client.Client
+	uncachedReader client.Reader
+	crdList        crd_watch.CrdList
```

Comment:
Sometimes vars of this type are called `crdWatch`, sometimes `crdList`. Stick to one to be consistent?

Reply:
ACK, that got me confused.